packages/grid/_modules_/grid/components/menu/columnMenu/GridColumnHeaderMenu.tsx

```diff
@@ -44,6 +44,9 @@ export function GridColumnHeaderMenu({
           apiRef!.current!.rootElementRef!.current!,
           field!,
         );
+        if (!headerCellEl) {
+          return;
+        }
```

Comment:
This branch looks strange. I would have expected no branch at all. If there is a good use case, shouldn't it branch earlier? For instance, does it fail because `apiRef!.current!.rootElementRef!.current` is not be defined yet?

Reply:
it will go anyway with #1460